core/ic/src/tera/src/api/messages.rs

```diff
@@ -19,6 +21,12 @@ fn get_messages() -> Vec<OutgoingMessagePair> {
     STATE.with(|s| s.get_messages())
 }
 
+#[update(name = "get_incoming_messages", guard = "is_authorized")]
+#[candid_method(update, rename = "get_incoming_messages")]
+fn get_incoming_messages() -> HashMap<String, u32> {
```

Comment:
I think defining `pub type MessagesCount = u32` might increase code readability

Reply:
Although I agree, this return type is predicated on an existing structure that wasn't newly introduced. Another PR would best suite this change. 

```
#[derive(CandidType, Deserialize)]
pub struct TerabetiaState {
    /// Incoming messages from L1
    pub messages: RefCell<HashMap<String, u32>>,
    
    ....
}
```